Code/Tools/SceneAPI/SceneBuilder/Importers/AssImpMaterialImporter.cpp

```diff
@@ -195,11 +196,18 @@ namespace AZ
                     return texturePathRelativeToScene;
                 }
 
+                bool ok;
+                AZStd::string relativePath, rootPath;
+                AzToolsFramework::AssetSystemRequestBus::BroadcastResult(
+                    ok, &AzToolsFramework::AssetSystemRequestBus::Events::GenerateRelativeSourcePath,
+                    texturePathRelativeToScene.c_str(), relativePath, rootPath);
+
                 // The engine only supports relative paths to scan folders.
-                // Scene files may have paths to textures, relative to the scene file. Check for those, first.
-                if (AZ::IO::FileIOBase::GetInstance()->Exists(texturePathRelativeToScene.c_str()))
+                // Scene files may have paths to textures, relative to the scene file.
+                // Try to use a scanfolder relative path instead
```

Comment:
The previous path generation logic was added because o3de\Gems\Atom\Feature\Common\Code\Source\Material\MaterialConverterSystemComponent.cpp's function MaterialConverterSystemComponent::ConvertMaterial, in the handleTexture lambda, relative paths were not working, it needed absolute paths. Does that work correctly now?

Reply:
I tested with one of the test assets and the materials show up fine.  I also stepped through the code, it is returning a valid path from the API call to GetSourceInfoBySourcePath.  If you look at the code for that API, it handles both abs and relative paths.  I think it probably just failed before from paths not being relative to a scanfolder.

Reply:
I tested with one of the test assets and the materials show up fine. I also stepped through the code, it is returning a valid path from the API call to GetSourceInfoBySourcePath. If you look at the code for that API, it handles both abs and relative paths. I think it probably just failed before from paths not being relative to a scanfolder.